base_conditional_image_for_partner/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = ["res.partner", "conditional.image.consumer.mixin"]
```

Comment:
We need to check this together. I think we have sides effects on the logo used on companies.

Reply:
Fixed in the new PR that superseed this one.